src/zenml/cli/stack_recipes.py

```diff
@@ -905,45 +960,52 @@ def deploy(
-                        f"list of outputs. To now retrieve sensitive "
-                        f"outputs, for example, the metadata-db-password "
-                        "use the command 'terraform output "
-                        "metadata-db-password' to get the "
+                        "\n Run 'terraform output' in the recipe's directory at "
+                        f"{local_stack_recipe.path} to get a list of outputs. To now "
+                        "retrieve sensitive outputs, for example, the metadata-db-password "
+                        "use the command 'terraform output metadata-db-password' to get the "
                         "value in the command-line."
                     )
 
         except RuntimeError as e:
             cli_utils.error(
                 f"Error running recipe {stack_recipe_name}: {str(e)} "
                 "\nPlease look at the error message to figure out why the "
-                "command failed. If the error is due some wrong "
-                "configuration, please consider checking the locals.tf "
-                "file to verify if the inputs are correct. Most commonly, "
-                "the command can fail due to a timeout error. In that "
-                "case, please run zenml stack recipe deploy "
-                f"{stack_recipe_name} again"
+                "command failed. If the error is due some wrong configuration, "
+                "please consider checking the locals.tf file to verify if the "
+                "inputs are correct. Most commonly, the command can fail due "
+                "to a timeout error. In that case, please"
+                f"run zenml stack recipe deploy {stack_recipe_name} again"
             )
         except python_terraform.TerraformCommandError as e:
             cli_utils.error(
                 f"Error running recipe {stack_recipe_name}: {str(e.err)} "
                 "\nPlease look at the error message to figure out why the "
-                "command failed. If the error is due some wrong "
-                "configuration, please consider checking the locals.tf "
-                "file to verify if the inputs are correct. Most commonly, "
-                "the command can fail due to a timeout error. In that "
-                "case, please run zenml stack recipe deploy "
-                f"{stack_recipe_name} again"
+                "command failed. If the error is due some wrong configuration, "
+                "please consider checking the locals.tf file to verify if the "
+                "inputs are correct. Most commonly, the command can fail due "
+                "to a timeout error. In that case, please"
+                f"run zenml stack recipe deploy {stack_recipe_name} again"
             )
 
 
+def zen_server_exists() -> bool:
+    """Check if a remote ZenServer is active.
+
+    Returns:
+        True if active, false otherwise.
+    """
+    gc = GlobalConfiguration()
+
+    return gc.store and gc.store.type == StoreType.REST
```

Comment:
I think this only checks if a ZenServer is used, but also returns True for a local (daemon or Docker) ZenServer.
```python
GlobalConfiguration().zen_store.is_local_store()
```
instead would give the correct result